Source/UI/NewUI/CommonUI/View/Core/TextFieldView.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+
+protocol TextFieldViewDelegate: AnyObject {
+    func textFieldShouldBeginEditing(textField: UITextField)
+    func textFieldShouldReturn()
+    func textFieldShouldEndEditing(textField: UITextField, replacementString: String)
+    func textFieldShouldChangeCharactersIn(textField: UITextField, replacementString string: String)
+}
+
+class TextFieldView: UIView {
+    weak var delegate: TextFieldViewDelegate?
```

Comment:
The `delegate` seems to never be set, but some callbacks are present. Is that expected?

Reply:
it's set 
`
 private(set) lazy var textFieldView: TextFieldView? = {
        let view = TextFieldView().disabledAutoresizingIntoConstraints()
        view.delegate = self
        return view
    }()
    `